app/views/publish/courses/_description_status_panel.html.erb

```diff
@@ -31,9 +31,7 @@
     <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
     <h3 class="govuk-heading-m govuk-visually-hidden">Delete</h3>
     <p class="govuk-body">
-      <%= govuk_link_to "Delete this course" %>
-
-      <%# govuk_link_to "Delete this course", delete_provider_recruitment_cycle_course_path(@provider.provider_code, course.recruitment_cycle_year, course.course_code), class: "app-link--destructive", data: { qa: "course__delete-link" } %>
+      <%= govuk_link_to "Delete this course", delete_publish_provider_recruitment_cycle_course_path(@provider.provider_code, course.recruitment_cycle_year, course.course_code), class: "app-link--destructive", data: { qa: "course__delete-link" } %>
```

Comment:
Can we have a token test to navigate there please

Reply:
done https://github.com/DFE-Digital/teacher-training-api/pull/2579/commits/d9e255c582894258b917c50fb72b2f1de2ea4257